habitat/datasets/rearrange/samplers.py

```diff
@@ -85,7 +85,6 @@ def __init__(
         self.receptacle_candidates: Optional[
             List[Receptacle]
         ] = None  # the specific receptacle instances relevant to this sampler
-        assert len(self.object_set) > 0
```

Comment:
Does it make sense to initialize an ObjectSampler with no valid object sets to sample from?